File: front-end/src/app/despesas/view-models/visualizar-despesa.view-model.ts
import { FormaPgtoDespesaEnum } from "./FormaPgtoDespesa.Enum";

export class VisualizarDespesaViewModel {
  id: string;
  descricao: string;
  valor: number;
  data: Date;
  formaPagamento: FormaPgtoDespesaEnum;
  categorias :string[] = []

}

File: back-end/eAgenad.WebApi/ViewModel/ModuloDespesa/VisualizarDespesaViewModel.cs
using eAgenad.WebApi.ViewModel.ModuloContato;
using eAgenda.Dominio.ModuloDespesa;
using System;
using System.Collections.Generic;

namespace eAgenda.Webapi.ViewModels
{
    public class VisualizarDespesaViewModel
    {
        public Guid Id { get; set; }
        public string Descricao { get; set; }

        public decimal Valor { get; set; }

        public DateTime Data { get; set; }

        public string FormaPagamento { get; set; }

        public List<string> Categorias { get; set; }
    }
}